lxqt-base/lxqt-config/files/kscreen-52690.patch

```diff
@@ -0,0 +1,80 @@
+From 6add4e4f0040693e7c4242fbae48c9d32007686c Mon Sep 17 00:00:00 2001
```

Comment:
Please include a link to the patch source upstream/wherever you got it from.

Reply:
done